### PR TITLE
Bartenders can use an emagged booze dispenser to create fake beer

### PR DESCRIPTION
**Pull request by sylvia-from-fulp-station**
## About The Pull Request

Emag a booze dispenser for fake beer.
Only people with the bartender's skill chip (TRAIT_DRUNKEN_BRAWLER) can emag a booze dispenser now.
QYUUymZQni

## Why It's Good For The Game

Traitors can get fake beer without needing a service borg.
Nobody really uses fake beer at the moment as its kind of ridiculous to flash, emag, job change and order a borg to dispense it.
Now you need to a. kill a bartender and steal his meridians, b. hypnotize or otherwise coerce a bartender into emagging it for you or c. be a bartender which is an otherwise rather underpowered role.

## Changelog

:cl:
qol: Booze dispensers that have had their safeties shorted by a Cryptographic Sequencer can now synthesize fake beer.
bal: You need to have the Bartender's skill chip to short the safeties of a booze dispenser.
/:cl:

### CODE REVIEW

**Comment by sylvia-from-fulp-station** [comment]
i accidentally left my map in my repo and removed it before pr plz dont kill me

**Comment by carlarctg** [comment]
the need to do teamwork and emag a service borg which is otherwise useless is the only thing that balances the instant-KO drug

**Comment by sylvia-from-fulp-station** [comment]
also u can get zombie powder with an emagged chem dispenser i dont really see how this is too different its still locked behind an emag, there is no teamwork envolved in either side ordering a borg to piss fake beer into a cup for 10mins in a welded maints room

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm
Comment (JohnFulpWillard) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm:625]:
```suggestion
		/datum/reagent/toxin/fakebeer,
```

**Review by MrMelbert — CHANGES_REQUESTED** [review]
I'm gonna say the chem needs to be tweaked for this to be merged. 

At the very least, make Fakebeer require ingestion for its instant sleep effect like Zombie powder.

It's trivially easy to make one of these in maint and fill up a bunch of syringes

**Comment by carlarctg** [comment]
i dont see why nerf the drug so pr goes through but w/e

**Comment by Exester509** [comment]
I'd much rather have this be a powerful chem that needs a special and riskier interaction than discount zombie powder from the bar.

**Comment by sylvia-from-fulp-station** [comment]
I don't think instant stun drugs are healthy for the game no matter what ridiculous obscure puzzle you put them behind

**Comment by MrMelbert** [comment]
what could they mean by this

**Comment by sylvia-from-fulp-station** [comment]
dont undraft this the job check does not work lol

**Comment by sylvia-from-fulp-station** [comment]
ok it works

https://github.com/user-attachments/assets/c2b8890a-4e9c-4997-98d2-a65ebd78da7d

**Comment by MrMelbert** [comment]
> I don't think instant stun drugs are healthy for the game no matter what ridiculous obscure puzzle you put them behind

uh, if you hate instant stun drugs, why are you making one more readily available?

**Comment by sylvia-from-fulp-station** [comment]
> uh, if you hate instant stun drugs, why are you making one more readily available?

Cuz i think its silly evil bartenders don't have access to an item called fake beer.
Also why the skill chip thing was what i went with.

in testing i discovered its not actually instant, still pretty fast but it doesn't really last that long. You can shoot people with HARS syringe as a gene tot and that is bad enough to make me ghost instantly and ive seen it more than fake beer.

If it being available to one job with antag gear on top of malf service borgs gets it nerfed then deserved ig

**Comment by the-orange-cow** [comment]
It would be much more interesting to keep the drug as is.  As stated above, it has a powerful but rare utility, if you nerf it then how is it any different from zombie poweder?

**Comment by sylvia-from-fulp-station** [comment]
The drug is as is, for this pr its just another way to get it.

**Comment by sylvia-from-fulp-station** [comment]
> So the whole logic behind this PR is because a regent named "fake beer" exists, it should be added to the emaged booze despenser? Maybe you should reconsider if you don't actually like the results of the change you are making.

i shouldn't even reply to this

The "whole logic behind this pr" is that this chem is underused. Ergo make it less annoying to access with similar requirements. Initially i overstepped forgetting people would just make a booze dispenser in maints and emag it as any job, but now i think it's fine. As a bartender you can use it with its intended purpose of pouring into a glass and serving as if it's beer and it might actually be used like this instead of by super power gamers who milk it from a service borg then make it into a foam grenade.

**Comment by Loafin34** [comment]
i think it's cooler for bartenders to be able to get it because borgs should not have a chokehold on this chem. You can just put 24u parapen mixture, 26u beer and it'll have the same effect, so why not?

**Comment by DrAmazing343** [comment]
I know we've had a precedent set with Roboticists emagging their exosuit fabs, but the emagged booze dispenser already exist so I don't want to PUT a lock on emagging it. Maybe make another option from the skillchip or something instead of locking another thing behind, but either way I'm not sure this is a greaaaat idea. B33R is super fucking scary.

**Comment by CCC23** [comment]
Why are people acting like a instant knockout chem is terrifying most times the moment you get hit with a chem you have about a half second to react before your already dead

**Comment by GuillaumePrata** [comment]
Having job checks on emag is a mistake that takes away from the sandbox and creativity.

More access to fake beer is a fun idea, but putting it on the booze dispenser feels like a noob trap.
The "secretive" evil beer leaves a big paper trail behind, anyone that checks the booze dispenser will know the bartender emaged the booze dispenser and is likely evil.

It would work a lot better IMO as a special shaker the bartender can buy, that transforms any booze into fake booze and then it can even have a bartender check to "use the secret compartment" or something and trigger the transformation, assassins teapot style.

**Comment by sylvia-from-fulp-station** [comment]
Yeah i don't really like locking stuff to a particular job but it's locked to their skill chip so it's possible to do it on any job. I don't like job locking stuff but i like the idea of people holding a bartender up at gunpoint to do it. Id argue that forcing a bartender to be involved is better for the sandbox; there is a challenge with multiple viable avenues to the same reward. the chems that were already on the emagged booze dispenser are quite useless and can already be obtained from vendors. I feel like this gives some much needed love to bartender tots but it seems like people are horrified of the fake beer chem.

**Comment by ExcessiveUseOfCobblestone** [comment]
> EDIT: A nerf is for a different pr. I made this pr mostly because i was convinced this interaction already existed in the game. Whether or not fake beer is a magic instant stun drug is for a different pr. i just think it should be available to tot bartenders and not a weird cyborg exclusive reagent.

It is not, or at least this will be blocked until that PR comes up so we can merge these in tandem. The accessibility requires the nerf, and the nerf requires the accessibility. They dont stand merge-able alone.

Please remove the job check for bartender or change this to not be an emag interaction (see https://github.com/tgstation/tgstation/pull/87090#issuecomment-2400363636 ).

> Why are people acting like a instant knockout chem is terrifying most times the moment you get hit with a chem you have about a half second to react before your already dead

It's not "terrifying", its just not fun to have literally zero counterplay. You can, with skill, react to a naughty syringe. a literally instant knockout is just not fun and the only excuse for it existing currently is because it is a pain to obtain which means it hasnt made the right person mad enough to change yet.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
see above comment

Basically needs the nerf to be considered, and the job exclusivity check removed (or taken off the emag, lady's choice)

**Comment by ExcessiveUseOfCobblestone** [comment]
shame

**Pull request closed without merging**
